runners/direct-java/pom.xml

```diff
@@ -83,7 +83,7 @@
               <failIfNoTests>true</failIfNoTests>
               <dependenciesToScan>
                 <dependency>org.apache.beam:beam-sdks-java-core</dependency>
-                <dependency>org.apache.beam:beam-runners-java-core</dependency>
```

Comment:
Instead of adding it to the test scope, we should limit this to adding it as plugin specific dependencies if possible.
https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Using_the_dependencies_Tag

This way it doesn't pollute the test scope of the regular project. We should follow this pattern for our other "integration" test setups.